practice/timezone/timezone/logic.py

```diff
@@ -0,0 +1,55 @@
+"""timezone Logic."""
+
+import arrow
+from tzwhere import tzwhere
+
+
+def get_current_time():
+    """Returns the current time in UTC formatted for ISO 8601."""
+    return arrow.utcnow().isoformat()
```

Comment:
What the exact string format of the response is is part of the view, so conversion to ISO format here isn't the job of the logic.